cli/tests/testdata/import_meta.ts

```diff
@@ -1,3 +1,9 @@
 console.log("import_meta", import.meta.url, import.meta.main);
 
 import "./import_meta2.ts";
+
+console.log("Resolving ./foo.js", import.meta.resolve("./foo.js"));
+console.log(
+  "Resolving ./foo.js from ./bar.js",
+  import.meta.resolve("./foo.js", "./bar.js"),
+);
```

Comment:
It could be worth including an import maps test as well here.

Also see the weird edge cases I posted in https://github.com/web-platform-tests/wpt/pull/34692/files if you're interested.

Reply:
There are also further tests at https://github.com/web-platform-tests/wpt/pull/34188/files.

Reply:
Thanks, I'll port the test using import map and enable remaining tests when they land in WPT suite.